test/common/formatter/substitution_formatter_test.cc

```diff
@@ -2213,7 +2213,7 @@ TEST(SubstitutionFormatterTest, CompositeFormatterSuccess) {
         "%%|%%123456000|1522796769%%123|1%%1522796769",
         formatter.format(request_header, response_header, response_trailer, stream_info, body));
   }
-#ifndef WIN32
+#if !defined(WIN32) && !defined(__APPLE__)
```

Comment:
Can you add a comment on why the ifdef is here on both platforms? I think some of the research you did would be helpful to the next person that comes along? Thank you.

/wait

Reply:
Yep -- doing some more local testing with Abseil  and then I'll push something.

Reply:
Done.